content/themis/installation/installation-from-sources.md

```diff
@@ -202,10 +193,10 @@ OpenSSL is the default cryptographic engine on most platforms.
 Themis will automatically detect and use OpenSSL installed in your system.
 If you wish to explicitly require OpenSSL, set `ENGINE=openssl` when building Themis.
 
-{{< hint warning >}}
+{{< hint info >}}
 **Note:**
-At the moment Themis supports only OpenSSL 1.0.2 and 1.1.x branches.
-OpenSSL 3.0 is still not fully supported.
+
+Starting from version [0.15.0](https://github.com/cossacklabs/themis/releases/tag/0.15.0), Themis supports OpenSSL 3.0. Consider using it instead of OpenSSL 1.1.1, as its end of life is [September 2023](https://www.openssl.org/blog/blog/2023/03/28/1.1.1-EOL/).
```

Comment:
let's mention that we tested with 3.0.2 and 3.0.8 if I'm not mistaken. Or can I ask @G1gg1L3s to look on openssl version in the packages of supported OSs in buildbot? which versions are installed to supported debian/ubuntu/centos that we test on buildbot?

Reply:
For now, only ubuntu:jammy has the openssl 3.0.2. Once we setup debian:bookworm, we would get openssl 3.0.9. So, yeah, let's put the tag on.